charts/opensrp-web/values.yaml

```diff
@@ -89,7 +89,7 @@ containerEnvironmentVariables:
   EXPRESS_OPENSRP_CLIENT_SECRET: "{{ .Values.sharedVars.opensrpClientSecret }}"
   EXPRESS_OPENSRP_LOGOUT_URL: "https://{{ .Values.sharedVars.opensrpServer  }}/opensrp/logout.do"
   EXPRESS_OPENSRP_OAUTH_STATE: "{{ .Values.express.opensrpOauthState }}"
-  EXPRESS_OPENSRP_USER_URL: "https://{{ .Values.sharedVars.opensrpServer  }}/opensrp/user-details/"
+  EXPRESS_OPENSRP_USER_URL: "{{ .Values.sharedVars.userDetailsUrl }}"
```

Comment:
For some reason this does not resolve correctly. This is what is generated in the server: `EXPRESS_OPENSRP_USER_URL=https://{{ .Values.sharedVars.keycloakUrl }}/auth/realms/{{ .Values.sharedVars.keycloakRealm }}/protocol/openid-connect/userinfo`

Reply:
let me check